src/client/src/orangehrmRecruitmentPlugin/components/CandidateProfile.vue

```diff
@@ -44,7 +44,7 @@
           <oxd-grid :cols="3" class="orangehrm-full-width-grid">
             <oxd-grid-item>
               <vacancy-dropdown
-                v-model="profile.vacancy"
+                v-model="vacancy"
```

Comment:
@SandeepaOHRM is this behaviour checked? Vacancy change confirmation message should appear.

![image](https://user-images.githubusercontent.com/12880059/176335863-9775f8b0-0178-45d3-9249-4282ef41fd0c.png)